e2e-tests/insurance/cypress/e2e/journeys/account/create/account-blocked/create-account-already-exists-and-is-blocked.spec.js

```diff
@@ -0,0 +1,29 @@
+import { INSURANCE_ROUTES as ROUTES } from '../../../../../../../constants/routes/insurance';
```

Comment:
Might need to update `test.yml`?

Reply:
already covered by `"account/create/**/*.spec.js",`